File: api/health.js
export default function handler(req, res) {
  res.setHeader("Access-Control-Allow-Origin", "*");
  res.setHeader("Access-Control-Allow-Methods", "GET, POST, OPTIONS");
  res.setHeader("Access-Control-Allow-Headers", "*");

  if (req.method === "OPTIONS") {
    return res.status(200).end();
  }

  return res.status(200).json({
    status: "online",
    service: "SkillBridge API",
    version: "2.0.0",
    tagline: "From Resume to Job-Ready.",
    hackathon: "Career Catalyst Club × GeeksforGeeks [CC-GFG-12]"
  });
}

File: api/health.py
from http.server import BaseHTTPRequestHandler
import json

class handler(BaseHTTPRequestHandler):
    def do_GET(self):
        self.send_response(200)
        self.send_header('Content-Type', 'application/json')
        self.send_header('Access-Control-Allow-Origin', '*')
        self.send_header('Access-Control-Allow-Methods', 'GET, OPTIONS')
        self.send_header('Access-Control-Allow-Headers', '*')
        self.end_headers()
        data = {
            "status": "online",
            "service": "SkillBridge API",
            "version": "2.0.0",
            "tagline": "From Resume to Job-Ready.",
            "hackathon": "Career Catalyst Club × GeeksforGeeks [CC-GFG-12]"
        }
        self.wfile.write(json.dumps(data).encode('utf-8'))
        return

    def do_OPTIONS(self):
        self.send_response(200)
        self.send_header('Access-Control-Allow-Origin', '*')
        self.send_header('Access-Control-Allow-Methods', 'GET, OPTIONS')
        self.send_header('Access-Control-Allow-Headers', '*')
        self.end_headers()
        return
